check-http/lib/check_http.go

```diff
@@ -30,6 +31,7 @@ type checkHTTPOpts struct {
 	Regexp             string   `short:"p" long:"pattern" description:"Expected pattern in the content"`
 	MaxRedirects       int      `long:"max-redirects" description:"Maximum number of redirects followed" default:"10"`
 	ConnectTos         []string `long:"connect-to" value-name:"HOST1:PORT1:HOST2:PORT2" description:"Request to HOST2:PORT2 instead of HOST1:PORT1"`
+	Proxy              string   `short:"x" long:"proxy" value-name:"[PROTOCOL://]HOST[:PORT]" description:"Use the specified proxy. PROTOCOL's default is http, and PORT's default is 1080."`
```

Comment:
[nit] Should add note that can pass `http://user:password@proxy.com` ? (but it's long-winded)

Reply:
Hmm, command line description is so narrow to describe everything, but adding `[USER:PASS@` seems acceptable.
Thank you for suggestion. I added at description (and tests for safe) at c92b8564a2a1d1df352e8d8bc4bf63a54ebb09af